duqtools/plot.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from duqtools.ids.ids_location import ImasLocation
+
+from .config import Config as cfg
+
+logger = logging.getLogger(__name__)
+info, debug = logger.info, logger.debug
+
+
+def plot(**kwargs):
+    """Plot subroutine to create plots from datas.
+
+    Parameters
+    ----------
+    kwargs :
+        kwargs
+    """
+
+    info('Extracting imas data')
+    # Gather all results and put them in a in-memory format
+    # (they should be small enough)
+    profiles = []
+    for entry in cfg().plot.data:
+        debug('Extracting database: %s, %s, %s, %s' %
+              (entry.db, entry.shot, entry.run, entry.user))
```

Comment:
I think you should be able to do this:

```suggestion
        debug('Extracting database: %s' % entry)
```